src/Command/Extension/SetExtensionVersionCommand.php

```diff
@@ -66,11 +70,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $versionReplacer = new VersionReplacer($version);
 
-        try {
-            $versionReplacer->setVersion($emConfFile, self::EMCONF_PATTERN);
-        } catch (\InvalidArgumentException $e) {
-            $io->error(sprintf('An error occurred while setting the ext_emconf.php version to %s.', $version));
-            return 1;
+        if ($dryRun) {
```

Comment:
Shouldn't this work exactly the opposite? In current state the command only works when having the `--dry-run` option set.

Reply:
Yeah, all conditions looks to be inverted. Not sure what I did here :(